tests/utils/Defaults.sol

```diff
@@ -33,7 +33,8 @@ contract Defaults is Constants, Merkle {
     //////////////////////////////////////////////////////////////////////////*/
 
     uint256 public constant AGGREGATE_AMOUNT = CLAIM_AMOUNT * RECIPIENT_COUNT;
-    string public constant CAMPAIGN_NAME = "Airdrop Campaign";
+    // Since Factory stores campaign name as bytes32, extra spaces are padded to it.
+    string public constant CAMPAIGN_NAME = "Airdrop Campaign                ";
```

Comment:
why not store it as `bytes32` also?

```solidity
bytes32 public constant CAMPAIGN_NAME_B32 = bytes32(abi.encodePacked(CAMPAIGN_NAME_S));
string public constant CAMPAIGN_NAME_S = "Airdrop Campaign";
```

Reply:
Because:
1. Its explicit this way i.e. clearly shows the padded whitespaces. By using `CAMPAIGN_NAME_B32`, it would not be obvious as to why we have to do that. 
2. We compare this string against string returned by factory function so CAMPAIGN_NAME_B32 is redundant.

Reply:
fair enough